### DISCUSSION TITLE
Request with Axios

### DISCUSSION THREAD

**Discussion by BetungasRamirezSatori** [alex8088/electron-vite#452] [Q&A] [answered]
I have been doing a project to test electron-vite, but when making requests with axios, all requests show me a blocked status
![image](https://github.com/alex8088/electron-vite/assets/134542857/01e78bdb-d81f-4d39-9f60-8b45e2288454)
![image](https://github.com/alex8088/electron-vite/assets/134542857/86c34b69-d8c0-43c5-b351-2d29be401963)

Do you have any solution for this?

**Comment by luckyLocas** [comment]
![image](https://github.com/alex8088/electron-vite/assets/23146290/e21d277e-01a8-40b5-8c7d-b846a8d37351)
Try setting webSecurity to false

**Reply by BetungasRamirezSatori** [reply]
![image](https://github.com/alex8088/electron-vite/assets/134542857/1ed83423-42f2-4633-a256-9546ee729e54)
I already have that flag but it still blocks me

**Comment by luckyLocas** [comment, marked as answer]
![image](https://github.com/alex8088/electron-vite/assets/23146290/df06b910-c742-4b78-b397-b1e52687d98b)
I just tested it and need to comment out this code

**Reply by BetungasRamirezSatori** [reply]
Thanks a lot, it worked for me!
